doc/nrf/ug_thread_tools.rst

```diff
@@ -37,22 +37,242 @@ Thread Border Router
+
+         #. Connect the nRF52840 Dongle to the USB port.
+         #. Press the **RESET** button on the dongle to put it into the DFU mode.
+            The LED on the dongle starts blinking red.
+         #. Install the RCP firmware package onto the dongle by running the following command, with */dev/ttyACM0* replaced with the device node name of your nRF52840 Dongle:
+
+            .. parsed-literal::
+               :class: highlight
+
+               nrfutil dfu usb-serial -pkg build/bin/ot-rcp.zip -p */dev/ttyACM0*
+
+      .. tab:: nRF52840 Development Kit (UART transport)
+
+         a. Flash the image using the nrfjprog utility:
+
+            .. code-block:: console
+
+               nrfjprog -f nrf52 --chiperase --program build/bin/ot-rcp.hex --reset
+
+         #. Disable the Mass Storage feature on the device, so that it does not interfere with the core RCP functionalities:
+
+            .. code-block:: console
+
+               JLinkExe -device NRF52840_XXAA -if SWD -speed 4000 -autoconnect 1 -SelectEmuBySN *SEGGER_ID*
+               J-Link>MSDDisable
+               Probe configured successfully.
+               J-Link>exit
+
+            Replace *SEGGER_ID* with the SEGGER ID of your nRF52840 Development Kit.
+            This setting remains valid even if you flash another firmware onto the device.
+         #. Power-cycle the device to apply the changes.
+
+Configuring using Docker
+========================
+
+For the development purposes, you can run the OpenThread Border Router on any Linux-based system using a Docker container that already has the Border Router installed.
+This solution is better when you are only interested in direct communication between your Border Router and the Thread network.
+For example, you can use the Docker container when you want to establish IP communication between an application running on Linux (such as Python Controller for Matter) and an application running on a Thread node.
+
+If you go for the Docker container option, make sure to :ref:`configure radio co-processor <ug_thread_tools_tbr_rcp>` first.
+
+To install and configure OpenThread Border Router on any Linux-based operating system using the Docker container, complete the following steps:
+
+#. Install the Docker daemon:
+
+   .. code-block:: console
+
+      sudo apt update && sudo apt install docker.io
+
+#. Start the Docker daemon:
+
+   .. code-block:: console
+
+      sudo systemctl start docker
+
+#. Create an IPv6 network for the OpenThread Border Router container in Docker:
+
+   .. code-block:: console
+
+      sudo docker network create --ipv6 --subnet fd11:db8:1::/64 -o com.docker.network.bridge.name=otbr0 otbr
+
+#. Download the latest version of the OpenThread Border Router Docker image by running the following command:
+
+   .. code-block:: console
+
+      docker pull openthread/otbr
+
+#. Connect the radio co-processor that you configured in :ref:`ug_thread_tools_tbr_rcp` to the Border Router device.
```

Comment:
Thus your Linux device?

Reply:
Correct. Do you want me to add this additional clarification?